src/room/InCallView.tsx

```diff
@@ -372,26 +374,27 @@ export function InCallView({
 
   if (noControls) {
     footer = null;
-  } else if (reducedControls) {
-    footer = (
-      <div className={styles.footer}>
-        <MicButton muted={microphoneMuted} onPress={toggleMicrophoneMuted} />
-        <VideoButton muted={localVideoMuted} onPress={toggleLocalVideoMuted} />
-        <HangupButton onPress={onLeave} />
-      </div>
-    );
   } else {
-    footer = (
-      <div className={styles.footer}>
-        <MicButton muted={microphoneMuted} onPress={toggleMicrophoneMuted} />
+    const buttons: JSX.Element[] = [];
+
+    if (!isOnlyScreenshare) {
```

Comment:
Didn't we still want to show the mic and video button but for them just to be disabled?

Reply:
The LocalFeed has no tracks. Unmuting and muting would results in an error. I'm catching this error in the SDK, but we decided to disable the buttons anyway. Actually, you would have to check whether an audio or video device is available. If this device is available we can display the buttons. But this is just a work around feature and I don't want ed change too much.

Reply:
The ideal solution from my PoV would be to display the buttons and clicking them causes it to try open the corresponding device again (I think this matches what other platforms do). This seems inconsistent since we don't hide the video button if you lack a webcam. We can ask a designer though.

Reply:
I tried to find a solution thats fits with the current behavior and software design. Currently this feature not change or touch old source code ore behaviour. 

I totally agree, this would be the best solution. But for this solution I need more time.

Reply:
Makes sense, my suggestion here would be to open an issue to fix this in the future, perhaps when we end up doing the refactoring